cachelib/allocator/CacheAllocator.h

```diff
@@ -2016,7 +2016,8 @@ class CacheAllocator : public CacheBase {
   // exposed for the background evictor to iterate through the memory and evict
   // in batch. This should improve insertion path for tiered memory config
   size_t traverseAndEvictItems(unsigned int tid, unsigned int pid, unsigned int cid, size_t batch) {
-auto& mmContainer = getMMContainer(tid, pid, cid);
+    util::LatencyTracker tracker{stats().bgEvictLatency_, batch};
```

Comment:
When you call tryEvictToNextMemoryTier on line 2049 (or tryPromoteToNextMemoryTier) it might call allocateInternal which from what I see will still use the regular latency tracker. This means that the background movement will still affect latency of allocation reported through `allocate.latency_us` and I think our goal was to only include latency of user-performance actions there.

Reply:
Added the fix in allocateInternalTier().